eio.opam

```diff
@@ -24,6 +24,7 @@ depends: [
   "alcotest" {>= "1.4.0" & with-test}
   "dscheck" {>= "0.1.0" & with-test}
   "odoc" {with-doc}
+  "yojson" {>= "2.0.2" & with-test}
```

Comment:
This change needs to go in dune-project. This is the cause of the CI error (though the opam lint is expected to fail anyway):
```
eio.opam: changed after 'dune build @install'!
```